samples/xsodata-simple/student.hdbtable

```diff
@@ -1,4 +1,4 @@
-table.schemaName = "PUBLIC";
+table.schemaName = "DBADMIN";
```

Comment:
people might change the user they're logging in with and don't have access to the `DBADMIN` schema. A separate `.hdbschema` would be best to avoid this

Reply:
+1